components/amorphic/lib/listen.js

```
@@ -42,15 +41,28 @@ function resolveVersions(packages) {
  * @param {unknown} sessionStore unknown
  * @param {unknown} preSessionInject unknown
  * @param {unknown} postSessionInject unknown
- * @param {unknown} sendToLogFunction unknown
+ * @param {unknown} logger unknown
  */
-function listen(appDirectory, sessionStore, preSessionInject, postSessionInject, sendToLogFunction, statsdClient, configStore = null) {
+function listen(appDirectory, sessionStore, preSessionInject, postSessionInject, logger, statsdClient, configStore = null) {
```

Since sendToLog function is no longer used, and if the consuming library still passes it, we should throw an error as we have deleted that logic completely. We can perhaps discuss?